dcap/types/src/certification_data.rs

```diff
@@ -132,13 +178,41 @@ mod test {
 
     extern crate alloc;
     use alloc::vec::Vec;
+    use yare::parameterized;
 
     #[test]
-    fn zero_certification_data() {
+    fn zero_type_is_error() {
         let bytes = [0u8; MIN_CERT_DATA_SIZE];
+        assert_eq!(
+            CertificationData::try_from(bytes.as_slice()),
+            Err(Quote3Error::CertificationDataType(0))
+        );
+    }
+
+    #[test]
+    fn type_eight_is_error() {
+        let mut bytes = [0u8; MIN_CERT_DATA_SIZE];
+        bytes[0] = 8;
+        assert_eq!(
+            CertificationData::try_from(bytes.as_slice()),
+            Err(Quote3Error::CertificationDataType(8))
+        );
```

Comment:
Same as above.

Reply:
used multi-line logic to separate out the `Err()`